18.04/php7.4/Dockerfile

```diff
@@ -104,16 +104,21 @@ RUN curl -L https://packagecloud.io/varnishcache/varnish41/gpgkey | APT_KEY_DONT
   && service varnish stop \
   && rm -rf /tmp/* /mysql-setup.sql 
 
-# Install Composer Latest Stable and Drush 10.x.
+# Install Composer Latest Stable and Drush 9.x.
 RUN curl -sS https://getcomposer.org/installer | php -- \
   --install-dir=/usr/local/bin \
   --filename=composer \
   && composer \
   --working-dir=/usr/local/src/ \
   global \
   require \
-  drush/drush:10.* \
-  && ln -s /usr/local/src/vendor/bin/drush /usr/bin/drush
+  drush/drush:9.* \
```

Comment:
Do we want to change this from 10.x?

Reply:
I backed this down because of:

> This assumes that your Drupal codebase is managed using Composer. The current recommendation is to install Drush on a per-project basis. This allows for having different versions of Drush installed for different projects. This is a best practice because different versions of Drush are compatible with a particular set of Drupal versions.

https://drupalize.me/tutorial/install-drush-using-composer?p=1156

Reply:
I read from Ultimike that Drush 10 is not supported on a global install also, and I have run into problems with it in some areas.

Reply:
Oh right. I forgot that is suggested to only be installed per project.

Perhaps we could set this up instead? https://github.com/drush-ops/drush-launcher

Reply:
We're still going to need the location of their vendor/bin directory. That is normally one directory above docroot. I think it needs to be done when environment variables are passed into the container via the probo app. Let me look into that.